src/Plugins/Hive/Drivers/OpenAi/Prompts/OpenAiImageAltPrompt.php

```
@@ -0,0 +1,83 @@
+use Thinktomorrow\Chief\Plugins\Hive\Drivers\OpenAiDriver;
+use Thinktomorrow\Chief\Sites\ChiefSites;
+
+class OpenAiImageAltPrompt implements HivePrompt
+{
+    use HivePromptDefaults;
+
+    private string $label = 'Verzin alt tekst voor deze afbeelding';
+
+    private array $altTexts = [];
+
+    public function getPayload(Field $field, ?string $locale, $livewireComponent): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$field'. (Unused Code Rules)

(UnusedFormalParameter)